client/state/index.js

```diff
@@ -31,6 +32,7 @@ import reader from './reader/reducer';
 export const reducer = combineReducers( {
 	plugins,
 	application,
+	document,
```

Comment:
I think we ought to rename this before merging. Let's go with `page` unless you have a better suggestion.